framework/include/mesh/MooseMesh.h

```diff
@@ -418,11 +418,35 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   ///@}
 
   /**
-   * Returns a map of boundaries to elements.
+   * Returns a map of boundaries to ids of elements on the boundary.
    */
   const std::unordered_map<boundary_id_type, std::unordered_set<dof_id_type>> &
   getBoundariesToElems() const;
 
+  /**
+   * Returns a map of boundaries to ids of elements on the boundary.
+   */
+  const std::unordered_map<boundary_id_type, std::unordered_set<dof_id_type>> &
+  getBoundariesToActiveLocalElemIds() const;
```

Comment:
Per discussion today
```suggestion
  const std::unordered_map<boundary_id_type, std::unordered_set<dof_id_type>> &
  getBoundariesToActiveSemiLocalElemIds() const;
```